tools/sync_stdlib_docs

```diff
@@ -25,7 +25,7 @@ fi
 #Removes a label, i.e a space, a variable name, followed by a colon followed by
 #an alphabetic character or ( or '. This should avoid altering the contents of
 #comments.
-LABREGEX="s/ [a-z_]+:([a-z\('])/ \1/g"
```

Comment:
This change is required to match `signal:Sys.signal` in `unixLabels.mli` previously there were no Module.type references.